src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,79 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final String SUPPLY = "supply";
+    private static final String BUY = "buy";
+    private static final int ONE_ARRAY_INDEX = 1;
+
     public void getStatistic(String fromFileName, String toFileName) {
+        String statistic = readFromFile(fromFileName);
+        writeToFile(toFileName, statistic);
+    }
+
+    private void writeToFile(String toFileName, String statistic) {
+        File myFile = new File(toFileName);
+        BufferedWriter bufferedWriter = null;
+        try {
+            bufferedWriter = new BufferedWriter(new FileWriter(myFile));
+            bufferedWriter.write(statistic);
+        } catch (IOException e) {
+            throw new RuntimeException("Can't write data to the file " + myFile, e);
+        } finally {
+            try {
+                if (bufferedWriter != null) {
+                    bufferedWriter.close();
+                }
+            } catch (IOException e) {
+                throw new RuntimeException("Can't close BufferedWriter", e);
+            }
+        }
```

Comment:
Consider using try-with-resources for `BufferedWriter` to ensure that the resource is closed automatically without a `finally` block.